modules/v2/variables.tf

```diff
@@ -308,3 +308,15 @@ variable "members" {
   description = "Users/SAs to be given invoker access to the service"
   default     = []
 }
+
+variable "create_service_account" {
+  type        = bool
+  description = "Create a new service account for cloud run service"
+  default     = true
```

Comment:
This is a breaking change, could we have a small upgrade guide about how to preserve historic behavior by setting `create_service_account=false`? Example: https://github.com/terraform-google-modules/terraform-google-bigquery/blob/master/docs/upgrading_to_bigquery_v8.0.md

Reply:
Done!